config_example.sh

```diff
@@ -104,6 +104,10 @@
 
 #export KUBERNETES_VERSION="v1.20.2"
 
+# Registry
+#export OVERRIDE_DOCKER_REGISTRY_1=${OVERRIDE_DOCKER_REGISTRY_1:-"docker.io"}
+#export OVERRIDE_DOCKER_REGISTRY_2=${OVERRIDE_DOCKER_REGISTRY_2:-"registry.hub.docker.com"}
```

Comment:
Those variables are really not explicit. Please do not use _1 and _2 but explicit names. In addition, the _2 seems unused ?

Reply:
renamed and removed _2.